src/replit/ai/modelfarm/__init__.py

```diff
@@ -1,6 +1,7 @@
 from .completion_model import CompletionModel as CompletionModel
 from .chat_model import ChatModel as ChatModel
 from .embedding_model import EmbeddingModel as EmbeddingModel
+from .txt2image_model import Txt2ImgModel as Txt2ImgModel
```

Comment:
Is this the name we want to go with? It seems a bit hard to parse compared to `Text2Image` or even better `ImageGenerationModel` (following the suite of `CompletionModel`, `ChatModel` and `EmbeddingModel`). OpenAI also calls this image generation instead of Text2Image.